react/week4/githubuser/src/UsersContext.js

```diff
@@ -0,0 +1,47 @@
+import React, { createContext, useState } from "react";
+
+export const UsersContext = createContext();
```

Comment:
This context is holding both the user state and the repos state, so I'd find a different name for this context

Reply:
yes i will change that to.